pexec/managed_process_test.go

```diff
@@ -182,7 +182,9 @@ func TestManagedProcessStart(t *testing.T) {
 
 			<-watcher.Events
 
+			test.That(t, proc.Status(), test.ShouldBeNil)
 			test.That(t, proc.Stop(), test.ShouldBeNil)
+			test.That(t, proc.Status(), test.ShouldNotBeNil)
```

Comment:
Can we assert anything about this error? What does it look like? I'm curious if we should be retuning an error at all if it's not a useful one, and whether `Status` should be named `Alive` and return a bool.

Reply:
My bad, I thought I added those changes. Though it may have been in the related PR draft in rdk. 

On error it can return things like 
- operation not permitted (when not owned(
- process does not exist
- process already exited (as in the test in the RDK pr)

Reply:
If you think the above errors don't add any value then yeah, I think Alive may be more intuitive

Reply:
Cool those error messages are def helpful;`Status` and returning an error seems correct, but let's assert that the messages are what we expect in tests here 👍🏻 .